prune-github-workflow-runs/package.json

```diff
@@ -0,0 +1,90 @@
+{
+  "name": "prune-github-workflow-runs",
```

Comment:
you have 2 lockfiles here - package-lock.jsoon and pnpm-lock.yaml. Decide on one package manager and then remove the other lockfile